awx_collection/tools/roles/template_galaxy/templates/galaxy.yml.j2

```
@@ -18,7 +18,7 @@ documentation: https://github.com/ansible/awx/blob/devel/awx_collection/README.m
 homepage: https://www.ansible.com/
 issues: https://github.com/ansible/awx/issues?q=is%3Aissue+label%3Acomponent%3Aawx_collection
 license:
-  - GPL-3.0-only
+  - GPL-3.0-or-later
```

The collection also contains `BSD-2-Clause` licensed files:

```suggestion
  - GPL-3.0-or-later
  # plugins/module_utils/tower_legacy.py
  - BSD-2-Clause
```

---

If I go to the commit that file came from and look in the folder where it lived:

https://github.com/ansible/ansible/tree/0e72f3bd71296bbb304e900296beb32e38cd44ab/lib/ansible/module_utils

_Most_ of the module_utils files look like they were usual GPL. But the first files in the folder, `_text.py` and `a10.py` were BSD. I feel fairly sure that the author from that time 7 years ago copied the header from one of those files and that was the extent of the legal logic at work.

---

module_utils in the ansible repo were required to be BSD licensed for a long time, it's more likely because of that.